Domains/3-DisciplineOther/StructuralAnalysis/StructuralComponents.md

```diff
@@ -0,0 +1,42 @@
+
+### StandardOrganizations Model
+
+#### Classification System
+
+"Standard Organizations" `DefinitionContainer` is broken down to a model that stores Elements that represents manuals. Manuals are defined as `ClassificationSystem` Elements where each Element represents only a single revision/edition of a catalog released, maintained by Standard Organization. `Code` value of the Manual element is expected to be concatination of StandardOrganization name and the manual name itself, e.g. `ClassificationSystem` that represents Standard Organization "CEN"'s "EU 19-53" Profile catalog edition would have a Code with value "CEN EU 19-53". `CodeScope` is expected to be set to `DictionaryModel` id, `CodeSpec` to `clsf:ClassificationSystem`.
+
+#### Classification Table
+
+The Manuals are expected to have child Elements (`ClassificationTable`) that divide the manual into groups by use case. e.g. Same manual might define both Beam Profiles and Steel Decks. The CodeScope of each `ClassificationTable` should be set to `ParentElement`. CodeSpec to "bis:ClassificationTable". Code values depend on the `ClassificationTable`'s use case:
+
+- Structural Profile Classifaction Table should have CodeValue set to "Structural Profiles"
+- Steel Decks to "Structural Steel Decks"
+- Joists to "Structural Joists"
+
+Each `ClassificationTable` is then broken down by a model that stores separate `Classifaction`s that belong to that table.
+
+#### Classifications
+
+`Classification` Elements define the actual entries of any Standard Catalog, e.g. AISC 7th Edition Structural Profile Catalog entires would be defined as separate `Classifications` containing all entries from that specific catalog edition.
+Each `Classifcation` CodeScope would be set to `Model`, each CodeSpec to `clsf:Classifcation`. The value would be set as Designation of that entry. (e.g. "W44X335"). `Classification` Elements would not define any properties that are defined by the actual real world catalog entries. Instead, such properties would be defined by separate Elements that reference classifications using `ElementHasClassifications` relationship.
+
+### Structural Components Catalog Definition Container
+
+Structural Compenents Catalog Hierrarchy defines actual structural components - Profiles, Steel Decks, Joists, etc... The top level Element of the hierrarchy is `DefinitionContainer` that is stored under `DictionaryModel`. The Code assigned to it consists of "Structural Components Catalog" value, `Repository` Model id scope, "bis:DefinitionContainer" spec. This container is broken down into case scpecific `DefinitionContainer` Elements - Profiles, Joists, Steel Decks, etc. Each container has a CodeScope set to "Structural Components Catalog" id, CodeSpec set to `bis:DefinitionContainer`. CodeValue depends on container's use case:
```

Comment:
The intention of this catalog was that it could be reused by other domains too. Would it make sense to name the top level container as "prf:DomainDefinitions"? Neither of the sub-models contains any StructuralAnalysis schema class element. All elements are defined by Profiles or Clasisfications schema classes.
I guess same rule would apply for "Standard Organizations" code

Reply:
Here, you're standardizing how those catalog-elements can be found by anyone. It has a two-level setup. The top-level element, since it is stored in the DictionaryModel, should follow the guidelines specified for them. Those guidelines aim at preventing any Code-collisions in the DictionaryModel (which is targeted by many data-writers). The second-level DefinitionContainers (e.g. Joists, Classifications, etc) don't necessarily need to follow such guidelines strictly since they are stored in a submodel that your domain is in control of.

Thus, it sounds like prf:DomainDefinitions is a more appropriate Code for the top-level element. You can follow the same convention for the second-level element-codes if you want, but it is not as critical. e.g. prf:StandardOrganizations.

Reply:
I've updated the documentation and added another top hierarchy Element "prf:DefinitionContainer"